deploy/template.yaml

```diff
@@ -123,6 +123,10 @@ objects:
             value: http
           - name: ASSISTED_SERVICE_HOST
             value: "assisted-service:8090"
+          - name: IMAGE_SERVICE_SCHEME
+            value: https
+          - name: IMAGE_SERVICE_HOST
+            value: api.integration.openshift.com/api/assisted-images
```

Comment:
Now that we're parsing this (to get the path) does it make sense to allow someone to specify the scheme as a part of this env var?

That would allow us to specify this as `IMAGE_SERVICE_HOST: ${OCM_BASE_URL}/api/assisted-images` in the saas file and not also specify the scheme.

We would need to implement some priority logic in the source to deal with if someone specifies `IMAGE_SERVICE_SCHEME` and also a scheme in the host env var, but that shouldn't be too bad.

Of course this will devolve into us not really needing the scheme env var, but I don't know that we want to go back and change all the other deployment methods now :tired_face: 

WDYT @osherdp @gamli75 @danielerez ?

Reply:
It won't be too clean to allow scheme in the host, but would indeed simplify the changes in saas file.
So maybe worth the change. I guess we could clean up the scheme var later from other deployments. I'll update the PR just to support it and we'll see how it goes.

Reply:
That all said, should we use the full URL here instead of splitting?

Also, this needs to be configurable. So list this in the parameters, put the default there and use the parameter here.

Reply:
What's written as IMAGE_SERVICE_HOST is not really the host part but rather host+path (sometimes called also URI)
I think we should have the full URL in one var, it will be easier to manage and work with

Reply:
Yeah, that's the issue we're dealing with here.
We forgot that `OCM_BASE_URL` and the additional path was something that we were going to have to deal with when deploying to the SaaS. So we're trying to deal with that while also not changing how we're deploying everywhere else.